transform/mattermost-analytics/models/marts/marketing/fct_marketing_campaign_member.sql

```diff
@@ -0,0 +1,38 @@
+with marketing as (
+    select marketing_id
+        , email as email
+        , server_id
+        , created_at
+        , updated_at
+    from
+        {{ ref('stg_cws__marketing')}}
+    where subscribed_content = 'security_newsletter'
```

Comment:
if we filter by "security_newsletter" shouldn't we name the CTE accordingly (as opposed to the generic "marketing")?

Reply:
Updated CTE & table name to reflect security_newsletter